pkg/network/namescheme/networknamescheme.go

```diff
@@ -20,37 +20,41 @@
 package namescheme
 
 import (
+	"crypto/md5"
```

Comment:
maybe consider using another algorithm ... md5 has been cryptographically broken and considered insecure.

Reply:
ok, even the [linter](https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/8899/pull-kubevirt-code-lint/1599799908020260864) frowns upon this ...
```
pkg/network/namescheme/networknamescheme.go:23:2: G501: Blocklisted import crypto/md5: weak cryptographic primitive (gosec)
	"crypto/md5"
	^
pkg/network/namescheme/networknamescheme.go:57:10: G401: Use of weak cryptographic primitive (gosec)
	hash := md5.New()
```

Reply:
Used sha256 instead.